ingestion_tools/dataset_configs/template.yaml

```diff
@@ -84,6 +84,15 @@ annotations: OPTIONAL
       mask_label: OPTIONAL, INTEGER (DEFAULT 1)
     parent_filters: see InstanceSegmentation.parent_filters
     exclude: SEE InstanceSegmentation.exclude
+  - TriangularMesh:
```

Comment:
@uermel Can you confirm that `TriangularMesh` is okay to refer to this annotation shape?

Reply:
✅ fine to call it Triangular mesh.